.yarnrc.yml

```diff
@@ -3,13 +3,10 @@ changesetBaseRefs:
   - upstream/main
   - origin/main
 
-initScope: uppy
+compressionLevel: mixed
 
 enableGlobalCache: false
-nodeLinker: node-modules
```

Comment:
Maybe let's not move this line around?

Reply:
This was automatically done by the upgrade. It doesn't matter either?